lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -273,6 +279,22 @@ type GetUTxOsStatistics = "wallets"
     :> "utxos"
     :> Get '[JSON] ApiUtxoStatistics
 
+{-------------------------------------------------------------------------------
+                                  Wallet Keys
+  See also: https://input-output-hk.github.io/cardano-wallet/api/#tag/WalletKeys
+-------------------------------------------------------------------------------}
+
+type WalletKeys =
+    GetWalletKey
+
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/getWalletKey
+type GetWalletKey = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "keys"
+    :> "script"
```

Comment:
What about capturing the full role here, so that we get better re-usability for this endpoint.